### ISSUE TITLE
Train model on custom actions

### ISSUE THREAD

**Issue by babdulhakim2** [FingerRec/real_time_video_action_recognition#2]
hello, Thank you for sharing your code, I've tested, and it's working though the accuracy is low of different video variations. I want to train custom video different action e.g smoking. I've got a video collection for this action, how do I go about the preprocessing and training?

Thanks

**Comment by FingerRec** [comment]
you can follow [c3d](https://github.com/hx173149/C3D-tensorflow) or [tsn](https://github.com/yjxiong/tsn-pytorch)
